DOCSP-52259-edit-aggregation-page (#123)
mongodb/docs-kotlin-sync: source/aggregation.txt

To learn more about Atlas Search pipeline stages, see :atlas:`Choose the
Aggregation Pipeline Stage </atlas-search/query-syntax/>` in the Atlas

> documentation. To view examples of Atlas Search pipeline search stages using the
> {+driver-short+}, see the :ref:`kotlin-sync-atlas-search` guide.

Reviewer: S: to avoid "using":
```suggestion
documentation. To view examples that use the {+driver-short+} to create Atlas Search pipeline search stages, see the :ref:`kotlin-sync-atlas-search` guide.
```
